src/devices/Amg88xx/InterruptModeBit.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Amg88xx
+{
+    /// <summary>
+    /// Defines the interrupt controls bits of the interrupt control register (addr: 0x03)
+    /// </summary>
+    public enum InterruptModeBit : byte
```

Comment:
Does this one needs to be public?
If yes, then use proper naming convention

Reply:
Can be internal, though renamed to InterrruptModeBits, too.
I use the InternalsVisibleToAttribute to expose it to the unit test assembly.